### PR TITLE
feat: Cron implementation

### PR DESCRIPTION
**Pull request by jonas-martinez**
Closes # 

## Checklist
- [ ] I didn't over-scope my PR
- [ ] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [ ] I added/updated the documentation about my changes
- [ ] I made my own code-review before requesting one

### CODE REVIEW

File: server/config/releases.exs [hunk 1]
```diff
@@ -10,6 +10,12 @@ config :dev_tools, DevTool.Repo,
   database: System.get_env("POSTGRES_DB"),
   hostname: System.get_env("POSTGRES_HOST")
 
+config :application_runner, ApplicationRunner.Repo,
```
Comment (taorepoara) [inline server/config/releases.exs:13]: You had to duplicate this config ?

Comment (jonas-martinez) [reply server/config/releases.exs:13]: Yes it seems that the config contained in application-runner is not applied. That is why I added the config here.

**Review by taorepoara — APPROVED** [review]

**Pull request merged**